buildpipeline/pipelines.json

```diff
@@ -366,7 +366,6 @@
           },
           "ReportingParameters": {
             "OperatingSystem": "Windows",
-            "Architecture": "arm64",
```

Comment:
These changes were not meant to be done. Looks like my local repo was in a weird state (and behind 13030). New commit should be correct